deployments/relations.ts

```diff
@@ -1,43 +1,59 @@
 import { RelationConfigMap } from '../plugins/deployment_manager/RelationConfig';
 
-let relationConfigMap: RelationConfigMap = {
+const relationConfigMap: RelationConfigMap = {
   comet: {
-    proxy: {
+    delegates: {
       field: {
         slot: '0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc',
       },
     },
     relations: {
-      cometAdmin: {
-        field: {
-          slot: '0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103',
-        },
-      },
       baseToken: {
-        alias: (token) => token.symbol(),
+        alias: async (token) => token.symbol(),
+      },
+      baseTokenPriceFeed: {
+        field: async (comet) => comet.baseTokenPriceFeed(),
+        alias: async (_, { baseToken }) => `${await baseToken[0].symbol()}:priceFeed`,
       },
       assets: {
         field: async (comet) => {
-          let assetLen = await comet.numAssets();
-          return await Promise.all(
-            [...new Array(assetLen)].map(async (el, i) => {
-              let assetInfo = await comet.getAssetInfo(i);
+          const n = await comet.numAssets();
+          return Promise.all(
+            Array(n).fill(0).map(async (_, i) => {
+              const assetInfo = await comet.getAssetInfo(i);
               return assetInfo.asset;
             })
           );
         },
-        alias: (token) => token.symbol(),
+        alias: async (token) => token.symbol(),
```

Comment:
Yeah this wasn't a meaningful change in itself, I ended up just making the style consistent through the file

Reply:
Yeah, I usually tend to do `async (x) => await y`, assuming that `(x) => y` and `async (x) => y` are all identical, since the first shows clearly that everything is promisey